{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Test defects","children":[{"name":"The following error originated from your application code, not from Cypress.\n\n  > The injectable 'PlatformLocation' needs to be compiled using the JIT compiler, but '@angular/compiler' is not available.\n\nThe injectable is part of a library that has been partially compiled.\nHowever, the Angular Linker has not processed the library such that JIT compilation is used as fallback.\n\nIdeally, the library is processed using the Angular Linker to become fully AOT compiled.\nAlternatively, the JIT compiler should be loaded by bootstrapping using '@angular/platform-browser-dynamic' or '@angular/platform-server',\nor manually provide the compiler with 'import \"@angular/compiler\";' before bootstrapping.\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","children":[{"name":"Check App is build and running","uid":"61c5b03de465671e","parentUid":"ccbac0977cf24caa609be85b5c338e7f","status":"broken","time":{"start":1698978903503,"stop":1698978903930,"duration":427},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":true,"retriesCount":3,"retriesStatusChange":true,"parameters":[],"tags":[]},{"name":"Check added user visible on both Apps","uid":"b70cf7b178fb6af1","parentUid":"ccbac0977cf24caa609be85b5c338e7f","status":"broken","time":{"start":1698978904480,"stop":1698978905000,"duration":520},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":true,"retriesCount":3,"retriesStatusChange":true,"parameters":[],"tags":[]}],"uid":"ccbac0977cf24caa609be85b5c338e7f"},{"name":"`cy.visit()` must be called with a `url` or an `options` object containing a `url` as its 1st argument\n\nhttps://on.cypress.io/visit","children":[{"name":"Check Angular App is running","uid":"25caf6c38b43191d","parentUid":"94cc8bfe4250392b21794aae9f4166bd","status":"broken","time":{"start":1698978959734,"stop":1698978960612,"duration":878},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":1,"retriesStatusChange":true,"parameters":[],"tags":[]}],"uid":"94cc8bfe4250392b21794aae9f4166bd"}],"uid":"bdbf199525818fae7a8651db9eafe741"},{"name":"Product defects","children":[{"name":"Timed out retrying after 10000ms: Expected to find element: `[data-e2e=\"FORM_FIELD__NAME\"]`, but never found it.","children":[{"name":"Add user to the table","uid":"a2367c617c232117","parentUid":"423749f86c4121661512fc2979efbc9c","status":"failed","time":{"start":1698979001926,"stop":1698979012925,"duration":10999},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":1,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"Remove user from the table","uid":"6147629bcc0f77cf","parentUid":"423749f86c4121661512fc2979efbc9c","status":"failed","time":{"start":1698979023810,"stop":1698979034585,"duration":10775},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":1,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"Check fields validation","uid":"76cf0fc41a74ff85","parentUid":"423749f86c4121661512fc2979efbc9c","status":"failed","time":{"start":1698979045401,"stop":1698979056153,"duration":10752},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":1,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"423749f86c4121661512fc2979efbc9c"},{"name":"Timed out retrying after 10000ms: Expected to find element: `h1`, but never found it.\n\nThe command was expected to run against origin `http://localhost:3002` but the application is at origin `about://blank`.\n\nThis commonly happens when you have either not navigated to the expected origin or have navigated away unexpectedly.","children":[{"name":"Check App1 and App2 elements","uid":"a46de58abe1862e3","parentUid":"9d8b4582ee49eef30e135438b0874a6d","status":"failed","time":{"start":1698979028159,"stop":1698979038823,"duration":10664},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":1,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"9d8b4582ee49eef30e135438b0874a6d"},{"name":"`cy.visit()` failed because you are attempting to visit a URL that is of a different origin.\n\nYou likely forgot to use `cy.origin()`:\n\n`cy.visit('http://localhost:3002/')`\n`<commands targeting http://localhost:3002 go here>`\n\n`cy.origin('http://localhost:3001', () => {`\n`  cy.visit('http://localhost:3001/')`\n`  <commands targeting http://localhost:3001 go here>`\n`})`\n\nThe new URL is considered a different origin because the following parts of the URL are different:\n\n  > port\n\nYou may only `cy.visit()` same-origin URLs within a single test.\n\nThe previous URL you visited was:\n\n  > 'http://localhost:3002'\n\nYou're attempting to visit this URL:\n\n  > 'http://localhost:3001'\n\nhttps://on.cypress.io/cannot-visit-different-origin-domain","children":[{"name":"Check Apps share div with React version 16.14.0","uid":"b6b4ca45e1b74dab","parentUid":"79d9ce04e5b4b4fb854060ab466e5da9","status":"failed","time":{"start":1698979098610,"stop":1698979099344,"duration":734},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"79d9ce04e5b4b4fb854060ab466e5da9"},{"name":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Cannot find module 'lodash'\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","children":[{"name":"Check header block with text visibility","uid":"5f1cac267a3f8f51","parentUid":"21f0a5b3e2f0770252f90d899aa78ff9","status":"failed","time":{"start":1698979110768,"stop":1698979111217,"duration":449},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":4,"retriesStatusChange":true,"parameters":[],"tags":[]},{"name":"Check button text visibility","uid":"de8b619a9ba92faa","parentUid":"21f0a5b3e2f0770252f90d899aa78ff9","status":"failed","time":{"start":1698979111549,"stop":1698979111846,"duration":297},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":5,"retriesStatusChange":true,"parameters":[],"tags":[]}],"uid":"21f0a5b3e2f0770252f90d899aa78ff9"},{"name":"Timed out retrying after 10000ms: Expected to find element: `.main`, but never found it.","children":[{"name":"Checks that both buttons are in the same block","uid":"761aa6bfa5339ca0","parentUid":"cdb14ca0edba907188ce5ec5cb56e932","status":"failed","time":{"start":1698979428092,"stop":1698979428092,"duration":0},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":3,"retriesStatusChange":true,"parameters":[],"tags":[]},{"name":"Checks that Webpack button stands as the first in the group","uid":"f27ea833cc45407","parentUid":"cdb14ca0edba907188ce5ec5cb56e932","status":"failed","time":{"start":1698979452604,"stop":1698979463163,"duration":10559},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":1,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"cdb14ca0edba907188ce5ec5cb56e932"},{"name":"Timed out retrying after 10000ms: Expected to find element: `.webpack-content div`, but never found it.","children":[{"name":"Checks buttons with text visibility","uid":"13d05737210fc41f","parentUid":"02aa8a1a0a9cbe0940e02605632b18d4","status":"failed","time":{"start":1698979428092,"stop":1698979428092,"duration":0},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":2,"retriesStatusChange":true,"parameters":[],"tags":[]}],"uid":"02aa8a1a0a9cbe0940e02605632b18d4"},{"name":"The following error originated from your test code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > window.confirm wasn't called within 1s\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.","children":[{"name":"Clicks on webpack content button and checks that wrong alert greeting is not displayed","uid":"32e8049170be75d3","parentUid":"fe45184aceb13f157d3c84e6378d0a3c","status":"failed","time":{"start":1698979431636,"stop":1698979435101,"duration":3465},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":1,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"Clicks on webpack content button and checks correct alert greeting","uid":"5a931d666e13aa5d","parentUid":"fe45184aceb13f157d3c84e6378d0a3c","status":"failed","time":{"start":1698979438543,"stop":1698979442128,"duration":3585},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":1,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"fe45184aceb13f157d3c84e6378d0a3c"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"}],"name":"categories"}